department-of-veterans-affairs/caseflow
# BEAAM Frontend

Connects #5631 

### Description
Added a new `queue/beaam` route which shows a list of all of the BEAAM appeals. The frontend is a modified version of the attorney queue.

### Acceptance Criteria 
- [ ] If `:queue_beaam_appeals` is enabled then a user should be able to go to `queue/beaam` and see all of the appeals in the new appeal table.

### Testing Plan
1. Add appeals to the Appeal table
1. Go to `queue/beaam` and ensure the appeals are listed
